core/package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "@uiw/react-md-editor",
-  "version": "3.23.6",
+  "version": "3.24.0",
   "description": "A markdown editor with preview, implemented with React.js and TypeScript.",
   "homepage": "https://uiwjs.github.io/react-md-editor/",
-  "author": "kenny wang <wowohoo@qq.com>",
+  "author": "kenny wang <wowohoo@qq.com>, RARgames",
```

Comment:
@RARgames If we need to add `Contributors` we should add the other 21 people

```
"contributors": [...]
```

If there is an `AUTHORS` file in the root of your package, npm will treat each line as a `Name <email> (url)` format, where email and url are optional. Lines which start with a # or are blank, will be ignored.

Reply:
No, I'll delete that (It was added just for my fork).